src/routes/_surfaces/workspace/explore/leaderboards/LeaderboardDisplay.svelte

```diff
@@ -104,6 +111,20 @@
   onDestroy(() => {
     observer?.disconnect();
   });
+
+  /** Filter out the leaderboards whose underlying dimensions do not pass the validation step. */
+  $: validDimensions =
```

Comment:
Let's also use `validDimensions` in `leaderboardFormatScale` instead of `$metricsExplorer.leaderboards`